src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/payment_instruments/savePayment.js

```diff
@@ -44,10 +44,13 @@ function getResponseBody(action) {
 }
 
 function savePayment(req, res, next) {
-  if (!AdyenHelper.getAdyenSecuredFieldsEnabled()) {
+  if (
+    [constants.METHOD_ADYEN_COMPONENT, 'ADYEN_CREDIT'].indexOf(
```

Comment:
Is there no constant we can declare for ADYEN_CREDIT?

Reply:
no, and actually I just noticed `constants.METHOD_ADYEN_COMPONENT` returns "AdyenComponent" whereas the expected payment processor ID is "Adyen_Component" so I can't use the constants here